app/services/licences/supplementary/submit-mark-for-supplementary-billing.service.js

```diff
@@ -65,20 +74,38 @@ async function _flagLicenceYears(supplementaryYears, licenceId, user) {
     })
   }
 
-  if (supplementaryYears.length === 0) {
-    return
+  let twoPartTariffBillingYears = []
+  const twoPartTariff = true
```

Comment:
As far as I can see this variable isn't used outside the scope of the `if (supplementaryYears.length > 0) { ... }` block so it could be moved into that.